.github/workflows/test.yml

```diff
@@ -32,6 +32,7 @@ jobs:
         run: nimble install -y
       - name: Run nim c -r tester
         run: |
+          nimble install
```

Comment:
```suggestion
```

Reply:
It needs `nimble install` for running the tests, no?

Reply:
> run: nimble install -y

My previous PR had added it already above